spectrocloud/schemas/cluster_location_config.go

```diff
@@ -0,0 +1,42 @@
+package schemas
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func ClusterLocationSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"country_code": {
+					Type:     schema.TypeString,
+					Optional: true,
+					Default:  "",
+				},
+				"country_name": {
+					Type:     schema.TypeString,
+					Optional: true,
+					Default:  "",
+				},
+				"region_code": {
+					Type:     schema.TypeString,
```

Comment:
```suggestion
					Type:     schema.TypeString,
					Description: "The region code of where the cluster is located in.",
```
Same as above, to we know what the ISO value is?